.github/workflows/run-tests.yml

```diff
@@ -27,9 +27,19 @@ jobs:
           path: ~/.npm
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: ${{ runner.os }}-node-
+      - name: Security audit
+        run: npm audit    
```

Comment:
I guess my only question is: are there ever vulns we cannot fix? Thinking about the possibility of a dependency having a vuln which doesn't have a fix yet and us not being able to release new versions 🤔 

Frontend for example fails npm audit, maybe we want to run this at the critical audit level `--audit-level critical`, or maybe we only want to run it on PRs and not merges 🤔 

As with my other comment I'm not asking for changes here, just something for us to think about as well as we move things over.

Reply:
Yeah... on some Jenkins jobs we had an optional checkbox for whether to run `npm audit` or not.

Is there likely to be a situation where a vuln passes the PR check but would fail on master (i.e. would not having the post-merge check make any difference)?